_data/en/settings.yml

```diff
@@ -185,10 +189,10 @@ nav:
   contact_us:
     title: Contact us
     sections:
-      - name: Get in Touch
+      - name: Find answers to commonly asked questions
```

Comment:
Was it intentional to reorder the page sections so that the FAQs are at the top of the Contact page?

Reply:
Yep, I believe that the idea is folks will look at the Q/A stuff first, and then they'll seek help if they don't see an answer to their question.

Reply:
Gotcha. As a related aside, is there a ticket or design reference I could consult when reviewing this? Including that would help with the background context.